package/src/components/ErrorsBlock/v1/ErrorsBlock.md

```diff
@@ -3,7 +3,9 @@
 The `ErrorsBlock` is used to provide helpful information to the user when a form is invalid. The block takes an array of errors that are displayed with an optional icon.
 
 ```jsx
-const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
+import Field from "../../Field/v1/Field"
+import TextInput from "../../TextInput/v1/TextInput"
+;const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
```

Comment:
Error here caused by `;const`: https://deploy-preview-421--stoic-hodgkin-c0179e.netlify.com/#/Base%20Components?id=errorsblock

The `;` semicolons got pushed down into the 2nd row for a lot of these. Putting them back on the same line as the `import` should fix these!